projects/verifiable-credential-wallet.md

```diff
@@ -0,0 +1,59 @@
+# Project Name
+Verifiable Credential Wallet
+
+# Preferred Maturity Level
+Labs stage
+
+# Project Description
+The Verifiable Credential Wallet is a cross-platform iOS and Android mobile application for storing and sharing digital credentials, starting with higher learning.
+
+The wallet is based on the learner credential wallet specification developed by the Digital Credentials Consortium. 
+
+The app works with iOS and Android and allows users to add and share credentials, as well as manage the wallet.
+
+Whenver possible, we use existing open standards from W3C, IETF, and other open standards groups.
+
+
+# Alignment with the OpenWallet Foundation Mission
+This project aligns with the OWF mission, because it's an open source example implementation of wallet-related open standards.
+
+# Code of Conduct
+https://wiki.dcconsortium.org/app/page/1SYYIdVWKAdBIBx3BwSFWZKytJ9zPprAZFTbD_7Bfo4Y?p=1h1VJHHv2zSe0n9Ltg-KQevSuqWbGcExy
```

Comment:
Will we keep this or adopt LF's one

https://linuxfoundation.eu/en/policies

Reply:
@skounis Just confirmed, we will adopt LF's.